packages/fxa-profile-server/lib/routing.js

```diff
@@ -84,6 +84,11 @@ module.exports = [
     path: v('/display_name'),
     config: require('./routes/display_name/post'),
   },
+  {
+    method: 'POST',
+    path: v('/display_name/auth/{uid}'),
```

Comment:
I guess we don't have a standard pattern to follow for auth-server to profile-server. We have used `/_` prefix and wonder if we could do that here.

Ex. `/_display_name/{uid}`

I think we should also leave a comment above to call out that this is an internal route only.